css/style.css

```diff
@@ -1,7 +1,16 @@
+* {
+  font-family: 'Poppins', sans-serif;
+
+  /* for check */
+
+  /* color: brown; */
+}
+
 body {
   padding: 0;
   margin: 0;
```

Comment:
- [ ] [Optional] I also suggest adding `margin: 0 auto;` since you have set the width. This centers your portfolio in the middle of the page for better viewing

Reply:
added
margin: 0 **auto**;